tests/test_activate.py

```diff
@@ -3357,3 +3357,18 @@ def test_stacking(create_stackable_envs, auto_stack, stack, run, expected):
         )
         == expected
     )
+
+
+def test_activate_and_deactivate_for_uninitialized_env(conda_cli):
+    # Call activate (with and without env argument) and check that the proper error shows up
+    assert "ERROR: Run 'conda init' before 'conda activate'" in conda_cli("activate")[0]
+    assert (
+        "ERROR: Run 'conda init' before 'conda activate'"
+        in conda_cli("activate", "env")[0]
+    )
+
+    # Call deactivate and check that the proper error shows up
+    assert (
+        "ERROR: Run 'conda init' before 'conda deactivate'"
+        in conda_cli("deactivate")[0]
+    )
```

Comment:
```suggestion
    assert "Run 'conda init' before 'conda activate'" in conda_cli("activate")[0]
    assert (
        "Run 'conda init' before 'conda activate'"
        in conda_cli("activate", "env")[0]
    )

    # Call deactivate and check that the proper error shows up
    assert (
        "Run 'conda init' before 'conda deactivate'"
        in conda_cli("deactivate")[0]
    )
```

Updating tests.

Reply:
I've updated the tests to check for a `CondaError`:
```python
    with pytest.raises(CondaError) as conda_error:
       conda_cli("activate")
    assert conda_error.value.message.startswith("Run 'conda init' before 'conda activate'")
```